packages/plugin/src/Library/DataObjects/Summary/Statistics/Spam.php

```diff
@@ -39,4 +39,10 @@ class Spam
 
     /** @var string */
     public $captchaType = '';
+
+    /** @var bool */
+    public $bypassSpamCheckOnLoggedInUsers = false;
```

Comment:
If you notice any places where the property type isn't defined - it's very welcome to specify them.

```suggestion
    public bool $bypassSpamCheckOnLoggedInUsers = false;
```